src/js/index.js

```diff
@@ -0,0 +1,50 @@
+import {
+  array, map, specialButton, specialCharacters,
+} from './globalVariables';
+import { CreateButtons } from './createButtons';
+import { tab } from './createSpecialButton/tab';
+import { space } from './createSpecialButton/space';
+import { backspace } from './createSpecialButton/backspace';
+import { enter } from './createSpecialButton/enter';
+import { shiftButtonDown } from './createSpecialButton/shiftButtonDown';
+import { shiftMouseDown } from './createSpecialButton/shiftMouseDown';
+import { capsLock } from './createSpecialButton/capsLock';
+import { changeKeyboardLayout } from './createSpecialButton/changeKeyboardLayout';
+import { outputValueKeydown } from './createSpecialButton/outputValueKeydown';
+import { outputValueMousedown } from './createSpecialButton/outputValueMousedown';
+import { del } from './createSpecialButton/del';
+import CreateKeyboard from './createKeyboard';
+
+const createKeyboard = new CreateKeyboard();
+const { textarea, keyboardContainer } = createKeyboard;
+createKeyboard.init();
+
+localStorage.setItem('first', '0');
+
+const createButtons = new CreateButtons(array);
+createButtons.crateButton(keyboardContainer);
```

Comment:
same as above:
better: `buttons` or `keyboardButtons`

also: this class `CreateButtons` is part of more general class `CreateKeyboard`
it should be initiated in `js/createKeyboard,js`